app/controllers/api/v3/top_profiles_controller.rb

```diff
@@ -0,0 +1,12 @@
+module Api
+  module V3
+    class TopProfilesController < ApiController
+      def index
+        @top_profiles = Api::V3::TopProfile.where(context_id: params[:context_id])
```

Comment:
let's not make context_id a required param

Reply:
👍